cms/templates/js/timed-examination-preference-editor.underscore

```diff
@@ -69,5 +69,16 @@
                 <p class='field-message' id='review-rules-description'><%- gettext('Specify any additional rules or rule exceptions that the proctoring review team should enforce when reviewing the videos. For example, you could specify that calculators are allowed.') %></p>
             </ul>
         </div>
+        <div class='timed-exam-options-list-fields is-hidden'>
+            <ul class="list-fields list-input timed-exam-options">
+                <li class="field-checkbox">
+                    <input type="checkbox" id="id_hide_exam_after_due_date" name="hide_after_due"  class="input input-checkbox"/>
+                    <label for="id_hide_after_due_date" class="label" aria-describedby="hide-after-due-description">
+                        <%- gettext('Hide Exam After Due Date') %>
+                    </label>
```

Comment:
If there is not already a Bok Choy accessibility test (that opens this dialog and clicks on the Advanced settings tab before verifying accessibility, please add one).